### PR TITLE
util/tracing: rename FormatRecordedSpans to Recording.String()

### PR DESCRIPTION
**Pull request by andreimatei**
Elevate the discoverability of FormatRecordedSpans() by making it the
stringer for a Recording. Remove the inferior stringer I had previously
added.

Release note: None

### CODE REVIEW

File: pkg/util/tracing/tracer_span.go [hunk 1]
```diff
@@ -266,6 +250,102 @@ func GetRecording(os opentracing.Span) Recording {
...
+
+func (l traceLogs) Less(i, j int) bool {
+	return l[i].Timestamp.Before(l[j].Timestamp)
+}
+
+func (l traceLogs) Swap(i, j int) {
+	l[i], l[j] = l[j], l[i]
+}
+
+// FormatRecordedSpans formats the given spans for human consumption, showing the
```
Comment (danhhz) [inline pkg/util/tracing/tracer_span.go]: s/FormatRecordedSpans/String/

**Review by danhhz — APPROVED** [review]

File: pkg/util/tracing/tracer_span.go (hunk 1 above)
Comment (andreimatei) [reply pkg/util/tracing/tracer_span.go]: done

**Review by RaduBerinde — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/41494#-:-LqqiRpEDIddjrSKcxjr:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @danhhz and @RaduBerinde)

**Review by andreimatei — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/41494#-:-LrF27sS-0THrnvnF5Uc:b-ydi9ay)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @danhhz)

**Review by andreimatei — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/41494#-:-LrQsi814Bc8dy-6ojXu:b-ydi9ay)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @danhhz)

**Pull request merged**